extensions/amp-ad-network-doubleclick-impl/amp-ad-network-doubleclick-impl-internal.md

```diff
@@ -125,3 +125,16 @@ Refresh is currently not supported for SRA enabled slots. If a slot is enabled f
 #### AMP Ad Container Compatibility
 
 The only AMP ad containers in which refresh is currently supported are amp-sticky-ad and amp-carousel container types.
+
+
+### SRA: Single Request Architecture (alpha)
+Enabling SRA allows a publisher to make a single request for all ad slots on the AMP page which gives a publisher the ability to do roadblocking and competitive exclusions. This very similar to the behavior acheived on non-AMP pages when using [this](https://developers.google.com/doubleclick-gpt/reference#googletag.PubAdsService_enableSingleRequest) method in GPT.
+
+In order to use this feature, add the following meta tag to the head of the AMP page:
+`<meta name=”amp-ad-doubleclick-sra”/>`
```

Comment:
Seems we allow arbitrary meta now: https://github.com/ampproject/amphtml/blob/e2f0f4c9126bac89a98e65722c8eede1a11b8285/validator/validator-main.protoascii#L501

Merging the PR now,  @Gregable let us know if this will have problem